addons/survey/views/survey_templates_statistics.xml

```diff
@@ -449,7 +447,7 @@
                                     t-field="choice_data['suggested_answer'].value_image"
                                     t-options='{"widget": "image", "qweb_img_responsive": False, "class": "o_image_64_max o_survey_answer_image", "alt-field": "name", "itemprop": "image"}'/>
                             </td>
-                            <td class="o_survey_answer">
+                            <td class="o_survey_answer text-nowrap">
```

Comment:
`py` to decrease row height?

Reply:
Reducing the row height by using `table-sm` like the matrix one and removing the `<p>` tag default margin bottom by adding `my-0`.